recipes/configure.rb

```diff
@@ -67,6 +67,20 @@
   notifies :restart, "service[#{node['nrpe']['service_name']}]"
 end
 
+if (platform?('fedora') && node['platform_version'].to_i >= 20) || (platform_family?('rhel') && node['platform_version'].to_i == 7)
```

Comment:
I think using "platform" and "platform_family" will trigger warnings on chef-12.
How about replacing this line with:

```
if node['init_package'] == "systemd"
```